hera_cal/nucal.py

```diff
@@ -507,21 +578,13 @@ def compute_spatial_filters(radial_reds, freqs, spatial_filter_half_width=1, eig
     # Create dictionary for all uv pswf eigenvectors
     spatial_filters = {}
 
-    # Get the minimum and maximum u-bounds used
-    u_bounds = get_u_bounds(radial_reds, radial_reds.antpos, freqs)
-
     # Loop through each baseline in each radial group
-    for gi, group in enumerate(radial_reds):
-        umin, umax = u_bounds[gi]
-        for bl in group:
-            umodes = radial_reds.baseline_lengths[bl] / SPEED_OF_LIGHT * freqs
-            pswf, _ = dspec.pswf_operator(
-                umodes, filter_centers=[0], filter_half_widths=[spatial_filter_half_width], eigenval_cutoff=[eigenval_cutoff], 
-                xmin=umin, xmax=umax, cache=cache
+    for group in radial_reds:
```

Comment:
Is this inefficient? Is there a circumstance where you'd be computing the same filters for different orientations? Or do we not really care?

Reply:
It may be a little, but the filters are dependent on baseline length and the difference between the minimum and maximum u of a particular group, so I was just assuming there would probably only be a few cases where you’d be recomputing the same filters. I have a speed-up that I have in mind for hera_filters that should speed this up by 2-3x anyway, so I don’t think this will be a huge bottleneck.